### ISSUE TITLE
How to help and timeframe for first release?

### ISSUE THREAD

**Issue by lightglitch** [sulu/SuluHeadlessBundle#6]
I'm interested in this bundle (mainly in the API for getting content, not react part for now), so instead on developing one from scratch how can I help? Do you have a list of tasks/todos?

I see that the website controller is not in the routes configuration, what arguments are expected?

**Comment by alexander-schranz** [comment]
@lightglitch the website controller will have no route. It is the controller you will use in your template. 

https://github.com/sulu/skeleton/blob/e6fded803d20e3e03e9ceb9b5e1fa15330928176/config/templates/pages/default.xml#L9

If you have a page `/en/example` the url `/en/example.json` will return you the headless data.

See also following graphic how it works:

![Headless - Rendering Process](https://user-images.githubusercontent.com/1698337/73056284-f7175100-3e8e-11ea-9e67-9371d8c65099.jpg)


Basically if you directly call `/en/example` we do not call `/en/example.json` as we write directly the data into the DOM and render it but all future requests will just call the `.json` of the pages you navigate to.

Follow the installation process on the README.md and please give us feedback how it works for you. Currently as you see in the issue content resolvers are missing and docs how it works, but it would be create to have feedback from the community how it works for you.

**Comment by nnatter** [comment]
Right now, the the timeframe for the first stable release heavily depends on external feedback. We are using the bundle in production and it feels very stable, but there might be edge cases that our use-case does not cover. 

Furthermore, as described in https://github.com/sulu/SuluHeadlessBundle/issues/3, the bundle does not cover every content-type right now.

If you want to support the development of the bundle, i see two ways to do so right now:
* Test the bundle in your project and give us feedback if it works for your usecase
* Implement the missing resolvers documented in https://github.com/sulu/SuluHeadlessBundle/issues/3

**Comment by alexander-schranz** [comment]
The `1.0.0` was released.
